packages/run-protocol/src/proposals/econ-behaviors.js

```diff
@@ -786,3 +797,17 @@ export const startRunStake = async (
   ]);
 };
 harden(startRunStake);
+
+/**
+ * @param {EconomyBootstrapPowers} space
+ */
+export const linkAmmAndReserve = async space => {
+  trace('linkAmmAndReserve');
+  const reserveInstance = await space.instance.consume.reserve;
+  trace({ reserveInstance });
+  const reservePublicFacet = await E(space.consume.zoe).getPublicFacet(
+    reserveInstance,
+  );
+  const ammCreatorFacet = await space.consume.ammCreatorFacet;
```

Comment:
These `await`s aren't necessary. I can imagine they were helpful for debugging... and I guess this isn't performance-sensitive stuff... so I guess we might as well leave them there to facilitate future debugging.

Reply:
exactly